index.js

```diff
@@ -0,0 +1,57 @@
+const cron = require('node-cron')
+const Logger = require('le_node')
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'));
+const {exec} = require('child_process');
+const formatter = require('dateformat')
+
+const CRON_TIME = process.env.CRON_TIME
+const MAX_BACKUPS = process.env.MAX_BACKUPS
```

Comment:
well, you use it only once. Does it really need a dedicated const? Especially that there are other env vars which you just use as `process.env....`